var/spack/repos/builtin/packages/gdal/package.py

```diff
@@ -131,9 +136,10 @@ class Gdal(AutotoolsPackage):
     depends_on('qhull', when='+qhull @2.1:')
     depends_on('opencl', when='+opencl')
     depends_on('poppler', when='+poppler')
+    depends_on('poppler@:0.63', when='@:2.3.0 +poppler')
```

Comment:
Looks like 2.3.1 fixed a bug in its poppler support:
https://trac.osgeo.org/gdal/wiki/Release/2.3.1-News